test/run/test.utils.js

```diff
@@ -35,14 +35,67 @@ describe("lib/utils", function () {
       return;
     }
 
-    // see ./mock-winreg.js
     var expected = "fake\\binary\\path";
+
+    // see ./mock-winreg.js
+    // Only mock keys in HKLM (local machine) hive
```

Comment:
I suppose that you changed the implementation to veify that the logic still works as intended when there is a HKLM-only registry key (i.e. without HKCU)? Can you clarify that intent in the comment, since it isn't obvious.

Reply:
That's right. If you have all the correct keys in HKLM and some missing in HKCU the intended behaviour (as I see it) is to ignore HKCU and fall back on HKLM for all the keys.

Reply:
I meant in a code comment, not here :)

Reply:
Haha, I thought it was a strange ask :P Will do.